internal/rollup/rollup_bundle.bzl

```diff
@@ -65,6 +65,13 @@ def write_rollup_config(ctx, plugins=[], root_dir=None, filename="_%s.rollup.con
   if not root_dir:
     root_dir = "/".join([ctx.bin_dir.path, build_file_dirname, ctx.label.name + ".es6"])
 
+  node_modules_path = ["."]
+  if ctx.attr.node_modules.label.workspace_root:
```

Comment:
nit: we have this logic in a bunch of other places, usually expressed as a const expression
```
"/".join[f for f in [workspace_root, package.split(), "thing"] if f]
```
would be nice to be consistent

Reply:
Thanks. Just thinking about this and although this logic works it assumes that the `node_modules` filegroup is defined in the same package as the `node_modules` folder is. Do you think this is reasonable restriction?

Reply:
Something like this from erickj's fix is more robust:

```
 node_paths = []
 for f in ctx.files.node_modules:
   # This is a bit of a hack, assuming the node_modules label is
   # actually referencing a directory called "node_modules".
   idx = f.path.find("node_modules")
   if idx >= 0:
      # Resolve paths relative to execroot.
      node_paths.append("./%snode_modules" % f.path[0:idx])
uniq_node_paths = {el: None for el in node_paths}.keys()
```

Or a similar approach is taken in this PR which just got put up:

https://github.com/bazelbuild/rules_nodejs/pull/176/files

Thoughts?

Reply:
Looks like this assumption is already made in rules_typescript as well
```
  config["bazelOptions"]["nodeModulesPrefix"] = "/".join([p for p in [
    ctx.attr.node_modules.label.workspace_root,
    ctx.attr.node_modules.label.package,
    "node_modules"
  ] if p])
```